aibolit/model/model.py

```diff
@@ -61,21 +23,19 @@ def preprocess_file(
             scale=False,
             **kwargs):
 
-        # df = pd.read_csv(r'D:\git\aibolit\scripts\target\dataset.csv')
-        print(Path(os.getcwd(), 'target', 'dataset.csv'))
-        df = pd.read_csv(str(Path(os.getcwd(), 'target', 'dataset.csv')))
+        df = pd.read_csv(Config.dataset_file())
         df = df[~df["filename"].str.lower().str.contains("test")]
 
         if self.do_rename_columns:
             p_codes = \
-                [x['code'] for x in CONFIG['patterns']] \
-                + ['lines' + x['code'] for x in CONFIG['patterns']]
-            m_codes = [x['code'] for x in CONFIG['metrics']]
+                [x['code'] for x in Config.get_patterns_config()['patterns']] \
```

Comment:
@lyriccoder Why not assigning `Config.get_patterns_config()` to local variable once and use the variable which would make the code cleaner?
```
config = Config.get_patterns_config()
....
[x['code'] for x in config['patterns']]
```